applications/games_menu/modules/menu.c

```diff
@@ -1488,9 +1489,9 @@ bool SavePresetGame(PresetStruct *preset)
 		if (FileExists(isoldr_preset_file_name))
 		{
 			fs_unlink(isoldr_preset_file_name);
-		}
+		}		
 		CopyFile(preset_file_name, isoldr_preset_file_name, 0);
-
+		
```

Comment:
Unnecessary changes

Reply:
Surely it was a mistake of the finger, I mean unintentionally hehe.
I'm going to delete it better, because the tabulator does it automatically.